Core/MIPS/ARM64/Arm64Asm.cpp

```diff
@@ -113,9 +115,109 @@ void Arm64Jit::GenerateFixedCode(const JitOptions &jo) {
 		loadStaticRegisters = nullptr;
 	}
 
-	enterCode = AlignCode16();
+	applyRoundingMode = AlignCode16();
```

Comment:
Because they are called by the dispatcher below. Need to know where it is before we can emit a function call to it.

Well, we could also patch it in afterwards but seems like unnecessary complication.